heron/executor/src/python/heron_executor.py

```diff
@@ -498,7 +498,7 @@ def _get_healthmgr_cmd(self):
                      "--cluster", self.cluster,
                      "--role", self.role,
                      "--environment", self.environment,
-                     "--topology_name", self.topology_name, "--verbose"]
+                     "--topology_name", self.topology_name]
```

Comment:
@ashvina - any reason why --verbose is removed?

Reply:
@kramasamy The hard-coded verbose was resulting in a noisy and big log files. I will replace it with the runtime config options, possibly using the dynamic config option @nwangtw has proposed in #2747

Reply:
i would suggest either,
keep the -verbose now till the code is stable
or,
align the -verbose with 'heron submit xxx -verbose': If user supplies verbose in cli, executor appends '-verbose'.